__tests__/app.test.js

```diff
@@ -76,4 +76,36 @@ describe("GET /api/articles/:article_id",()=>{
             expect(response.body.msg).toBe('Bad Request');
         });
     });
+})
+describe("GET /api/articles",()=>{
+    test("responds with status 200 and an articles array of article objects including total articles' comments, excluding body",()=>{
```

Comment:
I would just amend this test description to say that the articles will also be sorted by date in descending order. You may want to consider putting that functionality in a separate test if you feel it's a little long 🙂